src/bluesearch/database/article.py

```diff
@@ -393,6 +494,17 @@ def paragraphs(self) -> Generator[tuple[str, str], None, None]:
         for ref_entry in self.data["ref_entries"].values():
             yield "Caption", ref_entry["text"]
 
+    @property
+    def pmc_id(self) -> str | None:
+        """Get PMC ID.
+
+        Returns
+        -------
+        str or None
+            PMC ID if specified, otherwise None.
+        """
+        return self.data["paper_id"]
```

Comment:
Shouldn't we do `self.data.get("paper_id")` in case it is not there?

Reply:
You are right, I forgot to change it! Thanks for catching this. Done in 76fc0ef.